spec/models/message_spec.rb

```diff
@@ -124,13 +124,34 @@
     end
 
     describe 'ActivityNotification' do
-      subject { create(:message, request: request) }
+      subject { create(:message, :outbound, request: request) }
+
+      let!(:admin) { create(:user, admin: true) }
+
+      before do
+        organization.update!(users: create_list(:user, 5, organization: organization))
+      end
 
       it 'Message Received is not created for outbound messages' do
         expect { message }.not_to(change { ActivityNotification.where(type: 'MessageReceived').count })
       end
 
-      it_behaves_like 'an ActivityNotification', 'MessageReceived'
+      context 'But it is for inbound messages' do
+        subject { create(:message, :inbound, request: request) }
+
+        it 'creates a notification of type MessageReceived' do
+          expect { subject }.to change(ActivityNotification.where(type: 'MessageReceived'), :count).by(organization.users.count + User.admin.count)
+        end
+
+        it 'for each user and admin' do
```

Comment:
The description could do a better job on describing what's asserted here.

Reply:
See 704bc515